modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/props_transformer/cell_renderers/SharedItemRenderer.tsx

```diff
@@ -31,7 +31,18 @@ export default function SharedItemRenderer({
 	options: {actionId: string};
 	value: string;
 }) {
-	const {fileTypeIcon, fileTypeIconColor, siteName} = itemData;
+	const {assetType, fileTypeIcon, fileTypeIconColor, siteName} = itemData;
+
+	const [icon, iconColor] =
```

Comment:
This is fine, but using if/else conditional looks more readable to me, what do you think? 
```
let icon, iconColor;

if (fileTypeIcon && fileTypeIconColor) {
	icon = fileTypeIcon;
	iconColor = fileTypeIconColor;
} else if (assetType?.includes('Web Content')) {
	icon = 'forms';
	iconColor = 'content-icon-basic-content';
} else if (assetType?.includes('Blog')) {
	icon = 'blogs';
	iconColor = 'content-icon-blog';
} else if (assetType?.includes('Knowledge')) {
	icon = 'wiki';
	iconColor = 'content-icon-knowledge-base';
} else {
	icon = 'web-content';
	iconColor = 'content-icon-web-content';
}